DOCS-438 update Scaling-namespace-storage.md (#101)
aerospike/kubernetes-operator: docs/Scaling-namespace-storage.md

.
```

## Create a New Rack

> In order to resize `/dev/sdf` for namespace `test`  we have to create a new `rack` inside `rackConfig` with updated `storage` config and remove the old rack.

Reviewer: ```suggestion
To resize `/dev/sdf` for namespace `test`,  create a new `rack` inside `rackConfig` with updated `storage` config, and remove the old rack.
```

> The new rack can be created in the same physical rack using existing `zone/region` (if there is enough space) to hold new storage and old storage together.

Reviewer: ```suggestion
You can create the new rack in the same physical rack if there is enough space. Use the existing `zone/region` to hold the new storage and old storage together.
```

.
```

Save and exit the CR file, then use kubectl to apply the change.

```shell
kubectl apply -f aerospike-cluster.yaml
```

> This creates a new rack with `id: 2` and updated `storage` config. The old data will be migrated to new rack. The old rack will be removed gracefully.

Reviewer: ```suggestion
This creates a new rack with `id: 2` and updated `storage` config. The old data is migrated to new rack. The old rack is removed gracefully.
```
